CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Imperative package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Fixed a bug where the EDITOR environment variable was not being respected in a graphical environment [zowe/zowe-cli#1335](https://github.com/zowe/zowe-cli/issues/1335)
```

Comment:
Should we say `ZOWE_EDITOR`, or do we plan to only mention the full name in the CLI changelog?

Reply:
I was going to just say editor because the application could be different if it was implemented in a different app. It would make a lot more sense to say ZOWE_EDITOR in the CLI changelog. If you feel strongly that it should be ZOWE_EDITOR, I can change it.

Reply:
What about `<APP>_EDITOR` ?

Reply:
<APP> is not allowed by markdown without escape characters.